sql/dd/impl/upgrade/dd.cc

```diff
@@ -1031,6 +1032,124 @@ bool update_object_ids(THD *thd, const std::set<String_type> &create_set,
 }
 /* purecov: end */
 
+/*
+  Update myrocks internal data during dd_engine change from innodb to rocksdb
+
+  @param  thd                      Thread context.
+  @param  old_schema_name          Name of the old(from) schema
+  @param  new_schema_name          Name of the new(to) schema
+  @param  table_names              table name set to be updated
+
+  @returns false if success, true otherwise.
+*/
+bool update_myrocks_table_name(THD *thd, const String_type &old_schema_name,
```

Comment:
```suggestion
bool update_myrocks_table_names(THD *thd, const String_type &old_schema_name,
```
because it's multiple names in a set